test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -165,6 +182,52 @@ func describeQatDpdkPlugin() {
 			ginkgo.It("does nothing", func() {})
 		})
 	})
+
+	ginkgo.Context("When QAT's autoreset is off [Functionality:heartbeat]", func() {
+		ginkgo.BeforeEach(func() {
+			ginkgo.By("creating a configMap before plugin gets deployed")
+			e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "create", "configmap", "--from-literal", "qat.conf=AutoresetEnabled=off", "qat-config")
+		})
+
+		ginkgo.It("checks if the heartbeat is read correctly", func(ctx context.Context) {
+			injectError(ctx, f, nodeName)
```

Comment:
With the resource name known, we could search the nodeName before injecting the error?

Reply:
sorry, I didn't get your point.
Do you want me to remove the `nodeName` and make it searched in the injectError?

Reply:
After figuring out how to set two lines in one `qat.conf` configmap file, I thought it makes sense to remove the global variable `nodeName`, and put `resourceName` instead for this function so that can be found in the function `utils.FindNodeAndResourceCapacity`.